### PR TITLE
update README and fix some inaccessible links

### PR DESCRIPTION
**Pull request by bettybao1209**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,34 +40,43 @@ On Windows, use the [Neo version of LevelDB](https://github.com/neo-project/leve
...
 
-See also [official docs](http://docs.neo.org/en-us/node/introduction.html). Download and unzip [latest release](https://github.com/neo-project/neo-cli/releases).
+See also [official docs](https://docs.neo.org/docs/en-us/node/introduction.html). Download and unzip the [latest release](https://github.com/neo-project/neo-node/releases).
 
 ```sh
 dotnet neo-cli.dll
+
+or
+
+dotnet neo-gui.dll
```
Comment (erikzhang) [inline README.md]: This usage is not recommend.

**Comment by erikzhang** [comment]
It seems everyone is keen to modify the README.

**Comment by bettybao1209** [comment]
> It seems everyone is keen to modify the README.

File: README.md (hunk 1 above)
Comment (bettybao1209) [reply README.md]: Get it. I've spit them into two separate commands.

Comment (erikzhang) [reply README.md]: I mean, it is recommended to run it directly.

In linux:

```
./neo-cli
```

In windows, just double click the exe.

Comment (bettybao1209) [reply README.md]: thanks for the advice, just modified.

**Comment by erikzhang** [comment]
The logo should be updated too?

File: README.md [hunk 2]
```diff
@@ -38,34 +42,43 @@ On Windows, use the [Neo version of LevelDB](https://github.com/neo-project/leve
...
-cd neo-cli
+cd neo-node/neo-cli
 dotnet restore
 dotnet publish -c Release
 ```
-In order to run, you need .NET Core. Download the SDK [binary](https://www.microsoft.com/net/download/linux).
+Next, you should enter the `bin/Release/netcoreapp3.0` folder and paste the `libleveldb.dll` here. In addition, you need to create `Plugins` folder and put the `LevelDBStore` or `RocksDBStore` or other supported storage engine, as well as the configuration file, in the Plugins folder.
 
-Assuming you extracted .NET in the parent folder:
+Assuming you are in the `bin/Release/netcoreapp3.0` folder:
```
Comment (erikzhang) [inline README.md]: We should not write this kind of information that may change frequently in the README: `netcoreapp3.0`

Comment (bettybao1209) [reply README.md]: ok, I have removed it and used the working directory instead.

**Pull request merged**